javaslang/src/main/java/javaslang/collection/PriorityQueue.java

```diff
@@ -27,18 +26,18 @@
 public final class PriorityQueue<T> extends AbstractsQueue<T, PriorityQueue<T>> implements Serializable, Kind1<PriorityQueue<T>, T> {
     private static final long serialVersionUID = 1L;
 
-    private final SerializableComparator<? super T> comparator;
+    private final Comparator<? super T> comparator;
```

Comment:
We do it as in all other collections. In 3.0.0 we may change all APIs to accept a new/public SerializableComparator instead of a Comparator to ensure it is Serializable. This is a cross-cutting concern.